### ISSUE TITLE
Can be possible a ROS2 version for this project?

### ISSUE THREAD

**Issue by Ktysai** [code-iai/iai_kinect2#574]

**Comment by YuLiHN** [comment]
Hey, I just push a ROS2 version for kinect2, check it out here. https://github.com/YuLiHN/kinect2_ros2
It's buggy, still waiting to be completed but it is functional.

**Comment by eigendude** [comment]
I also ported to ROS 2 in 2022. Recently I started working on it again. Check it out here: https://github.com/eigendude/oasis_kinect2

**Comment by Helfaso** [comment]
@eigendude dose it work on depth topic now?

**Comment by eigendude** [comment]
No. not yet. I'm planning on finishing it later this year, but for now I'm just doing MediaPipe pose detection on the color image and using ORB_SLAM3 (which I've [also forked](https://github.com/eigendude/ORB_SLAM_OASIS)) for depth map generation.

If you know anyone working on the ROS 2 version of the driver in the meantime, I'm able to help with testing and collaboration.

**Comment by Helfaso** [comment]
i am work on ROS 2 to calibrate my Kinect v2 to use it on my robotic project, I am looking for a good toolbox for that so i can git the best out of the camera. 
i found this https://github.com/yukimakura/oasis_kinect2    repo forked from yours. have u tried it ,does it work????

**Comment by eigendude** [comment]
I found that before, but it only introduces a launchfile, and a package.xml fix which I applied.

**Comment by Abdulkareem771** [comment]
i have the same interest as @Helfaso has. i want to calibrate the camera initially using any tool even tools outside ros2 either on linux or windows. i just need the calibration file and i need it as accurate as possible. anyone knows a tool that can be in use for me in this case?

**Comment by Helfaso** [comment]
@eigendude so what toolbox do u recommended to do a full calibration??
